### DISCUSSION TITLE
Error 404 Code not found

### DISCUSSION THREAD

**Discussion by hearinghope** [vercel/community#6055] [Help]
### Summary

I am having trouble to resolve the issue as i am deploying the code on vercel and code is getting successfully deploy but it is showing error, I have also attached the screenshot for the reference.
![404-NOT_FOUND](https://github.com/vercel/community/assets/156460106/db4bccd4-d31f-4fbb-80f3-222bcf0adb78)

### Example

https://hearing-hope-test-repo.vercel.app/

### Steps to Reproduce

![404-NOT_FOUND](https://github.com/vercel/community/assets/156460106/86807440-900d-4848-afe0-fd10f2160db3)

**Comment by versecafe** [comment]
@hearinghope Just because the code is being deployed doesn't mean that the build outputs are correctly configured, what framework are you using and what is your deploy script?
